worker/src/main/java/gov/cms/ab2d/worker/processor/JobProcessorImpl.java

```diff
@@ -52,12 +56,13 @@
 @RequiredArgsConstructor
 @SuppressWarnings("PMD.TooManyStaticImports")
 public class JobProcessorImpl implements JobProcessor {
-    private static final String OUTPUT_FILE_SUFFIX = ".ndjson";
-    private static final String ERROR_FILE_SUFFIX = "_error.ndjson";
     private static final int SLEEP_DURATION = 250;
-    private static final long ROLL_OVER_THRESHOLD = 200; // 200 MB
-    private static final int ONE_MEGA_BYTE = 1024 * 1024;
 
+    @Value("${job.file.rollover.ndjson:200}")
+    private long ndjsonRollOver;
+
+    @Value("${job.file.rollover.zip:200}")
+    private long zipRollOver = 200;
```

Comment:
This initialization is probably redundant, since the property already declares the same default value.